packages/elements-core/src/hooks/useResizer.ts

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useRef, useState } from 'react';
+
+export function useResizer(sidebarWidth: number): [React.Ref<HTMLDivElement>, number, (event: any) => void] {
+  const sidebarRef = useRef<HTMLDivElement | null>(null);
```

Comment:
You are using `React.` prefix somewhere.
But here - don't.

Seems you need to align the approach